static/css/markers.css

```diff
@@ -0,0 +1,172 @@
+.gmnoprint[title^='דיון'] {
+    width: 30px !important;
+    height: 30px !important;
+    opacity: 1 !important;
+    -webkit-border-radius: 50% !important;
+    border-radius: 50% !important;
+    background-color: #5BC0EB;
+    transform: rotate(-45deg) translate(0px,-18px) !important;
+}
+
+.gmnoprint[title^='דיון']:after{
+    content: '\e0bf';
+    font-family: 'Material Icons';
+    font-size: 18px;
+    width: 24px;
+    height: 24px;
+    margin: 3px;
+    background: white;
+    display: inline-block;
+    -webkit-border-radius: 50%;
+    border-radius: 50%;
+    text-align: center;
+    line-height: 24px;
+    color: #5BC0EB;
+    -webkit-transform: rotate(45deg);
+    -moz-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+}
+
+.gmnoprint[title*='תאונה'],.gmnoprint[title*='תאונות']{
+    width: 30px !important;
+    height: 30px !important;
+    opacity: 1 !important;
+    -webkit-border-radius: 50% 50% 50% 0 !important;
+    border-radius: 50% 50% 50% 0 !important;
+    transform: rotate(-45deg) translate(12px,-31px) !important;
+}
+
+.gmnoprint[title*='תאונות']{
+    width: 30px !important;
+    height: 30px !important;
+    opacity: 1 !important;
+    -webkit-border-radius: 50%  !important;
+    border-radius: 50%  !important;
+    transform: rotate(-45deg) translate(12px,-31px) !important;
+}
+
+.gmnoprint[title*='תאונה']:after, .gmnoprint[title*='תאונות']:after {
+    font-family: 'Material Icons';
+    font-size: 18px;
+    width: 24px;
+    height: 24px;
+    margin: 3px;
+    background: white;
+    display: inline-block;
+    -webkit-border-radius: 50%;
+    border-radius: 50%;
+    text-align: center;
+    line-height: 24px;
+    -webkit-transform: rotate(45deg);
+    -moz-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+}
+
+.gmnoprint[title$='.']{
```

Comment:
I see what you mean about the title ending with a dot being the selector. There has to be a better way but this will do for now.